lib/postRunHook.js

```diff
@@ -0,0 +1,21 @@
+var Q = require('q');
+var events = require('./utils/events');
+var os = require('os');
+var logger = require('./utils/logger');
+
+module.exports = function(context) {
+
+    var options = context.opts;
+
+    // Check whether the livereload flag is present
+    if (!options.livereload && (options.options.indexOf('--livereload') === -1)) {
+        return Q();
+    }
+
+    // If the livereload flag is present, and we are running this function, 
+    // it means the server was successfully started.
+    // Therefores, let the user know how he can kill the LiveReload/BrowserSync server
+    var msg = 'LiveReload server running: ' + os.EOL + 'To kill it, please use: "CTRL+C" ' + os.EOL;
```

Comment:
There are lots of scenarios where you do not end up running the live reload server - if there are no platforms that are not supported, or error conditions etc. In those scenarios, are you sure that this message will not be logged? Perhaps you could set up some global variable using node's `globals` object and add a field to be true if live reload server is actually running and print this message only then.